client/extensions/woocommerce/state/sites/shipping-zone-locations/selectors.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { get, isObject } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { LOADING } from 'woocommerce/state/constants';
+
+const getRawShippingZoneLocations = ( state, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
If it is exported, I would use `getShippingZoneLocations`, maybe normalized to always return an array or object (rather than possibly returning `LOADING`). I think I suggested in another PR that the UI selector that returns things _with edits_ could be `get[Thing]WithEdits`